src/main/java/cz/centrum/haffner/SimpleTrainingDavid/AppServices/OneDayFileJsonParser.java

```diff
@@ -19,6 +21,11 @@
     private int okCallsCounter = 0;
     private int koCallsCounter = 0;
 
+    @Autowired
```

Comment:
Ty globální country, jejich nulování a pomocnou metodu pro jejich nulování smaž a udělej z nich lokální promenné.
Opět jsou to data, které se týkají každého souboru zvlášť - proto by se jejich scope měl týkat jen těch souborů.
Pro komponentu OneDayFileJsonParser jako takovou nemají žádné využití, proto je mít jako proměnné této komponenty nedává smysl.
Pokud důvod proč je mít globální a nulovat je při každém dalším zpracovaném souboru, je ten, aby se nemusely pokaždé znovu vytvářet... tak tohle vůbec neřeš - kompilátor si to po překladu do bytekódu optimalizuje takovým způsobem, že to nemá význam řešit.
Kromě toho : píšeš kód pro lidi, aby ho četli lidi - stroje si s tím poradí.

Reply:
předěláno na lokální proměnné ...